### PR TITLE
Update versions, includes fix of the build with new gradle plugin (junit 5 plugin).

### PR DESCRIPTION
**Pull request by yunikkk**

### CODE REVIEW

File: gradle/versions.gradle [hunk 1]
```diff
@@ -23,11 +23,11 @@ ext {
     mapbox_navigation = '0.42.5'
     mapbox_sdk = '9.0.0'
 
-    mannodermaus_junit5_version = '1.4.2.1'
+    mannodermaus_junit5_version = '1.6.0.0'
```
Comment (yunikkk) [inline gradle/versions.gradle:26]: @kiryldz this plugin caused `javac blabla` issues

File: gradle/versions.gradle [hunk 2]
```diff
@@ -3,7 +3,7 @@ ext {
     android_sdk_versions_plugin = '1.0.0'
 
     dokka_version = '0.9.17'
-    kotlin_version = '1.3.61'
+    kotlin_version = '1.3.72'
 
     min_sdk_version = 23
     target_sdk_version = 28
```
Comment (alexander-kulikovskii) [inline gradle/versions.gradle:9]: nit: 29
build_tools_version = 29.0.2

File: gradle/versions.gradle (hunk 1 above)
Comment (kiryldz) [reply gradle/versions.gradle:26]: Indeed that helps, thanks!

File: gradle/versions.gradle (hunk 2 above)
Comment (kiryldz) [reply gradle/versions.gradle]: Fixed.

Comment (yunikkk) [reply gradle/versions.gradle]: Well I haven't yet carefully reviewed implications on target sdk 29 https://developer.android.com/about/versions/10/behavior-changes-10

Comment (kiryldz) [reply gradle/versions.gradle]: I've merged it to dev because everything seemed to work at first glance.
I'll take a closer look tomorrow.

**Pull request merged**